boxoffice/static/js/models/sidebar.js

```diff
@@ -20,6 +20,12 @@ export const SideBarModel = {
           title: 'Dashboard',
           icon: 'fa-dashboard',
           view: 'dashboard'
+        },
+        {
+          url: '/ic/' + config.ic_id + '/orders',
```

Comment:
This hardcodes boxoffice to be hosted at `/`, preventing it from being hosted at `hasgeek.com/boxoffice` in future.